engine/src/flutter/lib/web_ui/lib/src/engine/semantics/table.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'label_and_value.dart';
+import 'semantics.dart';
+
+/// Indicates a table element.
+///
+/// Uses aria tab role to convey this semantic information to the element.
+///
+/// Screen-readers takes advantage of "aria-label" to describe the visual.
+class SemanticTable extends SemanticRole {
+  SemanticTable(SemanticsObject semanticsObject)
+    : super.withBasics(
+        EngineSemanticsRole.table,
+        semanticsObject,
+        preferredLabelRepresentation: LabelRepresentation.ariaLabel,
+      ) {
+    setAriaRole('tabble');
```

Comment:
s/tabble/table/

Reply:
```suggestion
    setAriaRole('table');
```

@yjbanov – it's cleaner 😀